main.go

```diff
@@ -30,28 +44,19 @@ func main() {
 	taskSvc := Tservice.New(taskStore, userSvc)
 	taskHandler := Thandler.New(taskSvc)
 
-	http.HandleFunc("GET /task", taskHandler.Viewtask)
-	http.HandleFunc("GET /task/{id}", taskHandler.Gettask)
-	http.HandleFunc("POST /task", taskHandler.Addtask)
-	http.HandleFunc("PUT /task/{id}", taskHandler.Updatetask)
-	http.HandleFunc("DELETE /task/{id}", taskHandler.Deletetask)
-
-	http.HandleFunc("GET /user", userHandler.Viewuser)
-	http.HandleFunc("GET /user/{id}", userHandler.GetUserByID)
-	http.HandleFunc("POST /user", userHandler.AddUser)
-
-	server := &http.Server{
-		Addr:         ":8080",
-		Handler:      nil,
-		ReadTimeout:  5 * time.Second,
-		WriteTimeout: 10 * time.Second,
-		IdleTimeout:  15 * time.Second,
-	}
+	app.GET("/", func(ctx *gofr.Context) (any, error) {
```

Comment:
We don't need the root route, but if we are using a root route. We can display a better message, like what routes are available.